app/services/manifest_builder/manifest_service_locator_v3.rb

```diff
@@ -70,6 +70,36 @@ def canvas_builder
           iiif_annotation_page_factory: iiif_annotation_page_factory
         )
       end
+
+      def shallow_child_manifest_builder_factory
+        IIIFManifest::ManifestBuilder::ChildManifestBuilderFactory.new(
+          composite_builder: composite_builder,
+          child_manifest_builder: child_manifest_builder
+        )
+      end
+
+      def child_manifest_builders
+        composite_builder_factory.new(
+          shallow_child_manifest_builder_factory,
+          structure_builder,
```

Comment:
When attempting an appraoch where `Collections` are generated (as opposed to `Manifests`) for multi-volume `Recordings`, this needs to be injected into the `child_manifest_builder` used to generate the Manifests in https://github.com/samvera-labs/iiif_manifest/blob/master/lib/iiif_manifest/manifest_builder/child_manifest_builder_factory.rb#L13.  Otherwise the `Ranges` within each member `Manifest` level simply aren't generated.